From e489fd04d66000829f5458843970794eccacced8 Mon Sep 17 00:00:00 2001
From: Eric Blake <ebb9@byu.net>
Date: Sat, 16 Jan 2010 13:46:17 +0000
Subject: build: fix failure from bogus USE_XATTR definition

* m4/xattr.m4 (gl_FUNC_ADDR): Fix regression introduced in commit
6beca4248.
* THANKS: Update.
Reported by Adam Sampson.
---
diff --git a/THANKS b/THANKS
index 583ea60..1207368 100644
--- a/THANKS
+++ b/THANKS
@@ -12,6 +12,7 @@ Aaron Hawley                        ashawley@uvm.edu
 Achim Blumensath                    blume@corona.oche.de
 Adam Jimerson                       vendion@charter.net
 Adam Klein                          aklein@debian.org
+Adam Sampson                        ats@offog.org
 Adrian Bunk                         bunk@stusta.de
 AIDA Shinra                         shinra@j10n.org
 Akim Demaille                       demaille@inf.enst.fr
diff --git a/m4/xattr.m4 b/m4/xattr.m4
index bf7e872..80fddbd 100644
--- a/m4/xattr.m4
+++ b/m4/xattr.m4
@@ -1,4 +1,5 @@
 # xattr.m4 - check for Extended Attributes (Linux)
+# serial 3
 
 # Copyright (C) 2003, 2008-2010 Free Software Foundation, Inc.
 # This file is free software; the Free Software Foundation
@@ -33,11 +34,11 @@ AC_DEFUN([gl_FUNC_XATTR],
         use_xattr=yes
       fi
     fi
-    AC_DEFINE_UNQUOTED([USE_XATTR], [$use_xattr],
-                       [Define if you want extended attribute support.])
     if test $use_xattr = no; then
       AC_MSG_WARN([libattr development library was not found or not usable.])
       AC_MSG_WARN([AC_PACKAGE_NAME will be built without xattr support.])
     fi
   fi
+  AC_DEFINE_UNQUOTED([USE_XATTR], [`test $use_xattr != yes; echo $?`],
+                     [Define if you want extended attribute support.])
 ])
--
cgit v0.8.2.1
